ipaserver/plugins/schema.py

```diff
@@ -214,10 +214,10 @@ class command(metaobject):
         ),
     )
 
-    def _iter_params(self, cmd):
```

Comment:
I get the reasoning for this but for some of these the context of the name change makes it easier to understand IMHO.

Would it be reasonable to do something like cmd = metaobj and leave the code the same? Or would that be even worse?

Reply:
> Would it be reasonable to do something like cmd = metaobj and leave the code the same? Or would that be even worse?

Particularly in this case `_iter_params` is overriden in class `command`, so, I expect that `metaobj` is somehow related to `command` even if I don't know internals. That's why I think this is redundant, but I can put some comment / sphinx markup / type hint, would it improve readability?

Reply:
Not worth it I think. If someone gets confused we can address it I guess.